Remove Git Fundamentals packages from automation

ldennington opened the pull request:
Removing `Microsoft.Git`, `Microsoft.GitCredentialManagerCore`, and
`Microsoft.VfsforGit` from automation. The Git Fundamentals team is
transitioning to using the `microsoft/wingetcreate` tool to generate
manifests for these tools and submit them to `microsoft/winget-pkgs`.

vedantmgoyal9 reviewed (changes requested):
@ldennington I do not delete JSONs to keep records. I have merged #48 which skips the packages.

vedantmgoyal9 committed:
Remove Git Fundamentals packages from automation

Removing `Microsoft.Git`, `Microsoft.GitCredentialManagerCore`, and
`Microsoft.VfsforGit` from automation. The Git Fundamentals team is
transitioning to using the `microsoft/wingetcreate` tool to generate
manifests for these tools and submit them to `microsoft/winget-pkgs`.
